### DISCUSSION TITLE
Does "rebase" work on the administration schedule or the configured schedule?

### DISCUSSION THREAD

**Discussion by andygnewman** [renovatebot/renovate#8519] [Request Help] [answered]
We have set up renovate to check our repos on a specific schedule, eg. monthly.

If we set `"rebaseWhen": "behind-base-branch"`, will this run on the monthly schedule that we configured for our repositories or the hourly schedule that it set for renovate within our organisation by the administrator?

I don't see any clarity on this within the documentation;
https://docs.renovatebot.com/updating-rebasing/
https://docs.renovatebot.com/configuration-options/#rebasewhen

**Comment by viceice** [comment, marked as answer]
check https://docs.renovatebot.com/configuration-options/#updatenotscheduled
